app/questionnaire/router.py

```diff
@@ -54,7 +56,7 @@ def get_next_location(self, location, routing_path):
 
         return routing_path[location_index + 1]
 
-    def get_previous_location(self, location, routing_path):
+    def get_previous_location_url(self, location, routing_path):
```

Comment:
This is now inconsistent with `get_next_location`.

Reply:
Both of them now return the url. `get_next_location ` -> `get_next_location_url`.